app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -42,9 +43,8 @@ echo "#"
 if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     CONSUMER_RPM=$(mktemp --suffix .rpm)
 
-    curl --insecure --output $CONSUMER_RPM <%= subscription_manager_configuration_url %>
+    curl --output $CONSUMER_RPM <%= subscription_manager_configuration_url %>
```

Comment:
this is the place we'll need to inject incoming url parameter as a prefix to subscription_manager_configuration_url if it's present, it will need katello PR to be merged first

Reply:
But there isn't a single URL. That's my whole point that I've been trying to make. There are 2 URLs:
* Foreman Proxy's `template_url` (typically a service on port 8000 over http)
* RPM URL (typically a service on port 80 over http)

If you pass in the URL parameter, you'll hit a HTTP 404 and won't work.

Reply:
After conversation, I think we came to an agreement, that we should be using SmartProxy (content_source) url, as it's URL that Foreman uses and has no guarantee, client can use that. Instead we'll rely on the url parameter being added here and we'll trust the proxy will send the best url possible (not based on template_url). We will still use subscription_manager_configuration_url to get correct `http` and everything, but the FQDN will be based on incoming url parameter.